3rdparty/orbbecsdk/orbbecsdk.cmake

```diff
@@ -0,0 +1,34 @@
+function(download_orbbec_sdk root_var)
```

Comment:
It's bad idea. SDK should be vendored in 3rdparty or pointed by CMake.
Reasons:
- Git pulled version is unpredictable.
- On-the-go download from Github that may be not available during build
- Pollute OpenCv repo during development.

Reply:
You can use `ocv_download` for version. Example:

https://github.com/opencv/opencv/blob/bfad61f43344f5d84bef86617bd4c1bacec0fe94/3rdparty/libtim-vx/tim-vx.cmake#L14-L20